kolibri/core/discovery/api.py

```diff
@@ -20,3 +22,18 @@ class DynamicNetworkLocationViewSet(NetworkLocationViewSet):
 
 class StaticNetworkLocationViewSet(NetworkLocationViewSet):
     queryset = StaticNetworkLocation.objects.all()
+
+
+class NetworkLocationFacilitiesView(viewsets.GenericViewSet):
+    """
+    Given a NetworkLocation ID, returns a list of Facilities that are on
+    that NetworkLocation, for the purposes of syncing
+    """
+
+    def retrieve(self, request, pk=None):
+        base_url = "http://192.168.1.8:8000/"
```

Comment:
Using a hardcoded ip is correct?

Reply:
No, this endpoint is still being worked on. It will use the `DynamicNetworkLocation` model's `baseurl` field to construct the `/api/public/v1/facility` endpoint for a specific peer.

This hardcoded URL is just my second computer's IP on the LAN, which I'm using for testing.